bucket.go

```diff
@@ -27,6 +27,13 @@ type BucketDocument struct {
 	Expiry uint32 // Item expiration time (UNIX Epoch time)
 }
 
+type Feature int
```

Comment:
Rename to something more specific - maybe BucketFeature?  (along with constants below)